src/foam/nanos/nanos.js

```diff
@@ -448,5 +448,8 @@ FOAM_FILES([
   { name: "foam/nanos/geocode/GoogleMapsCredentials" },
 
   { name: "com/google/foam/demos/u2/AllViews" },
-  { name: "com/google/foam/demos/u2/MDAllViews" }
+  { name: "com/google/foam/demos/u2/MDAllViews" },
```

Comment:
@kgrgreer can we move this foam/u2 ? 
@minsunny what in this are we using?

Reply:
this was added with this PR https://github.com/foam-framework/foam2/pull/4114

Reply:
My point here is that we are dependent on a 'demo' path for production code.  
If there is something in MDAllViews we are using it should move to foam/u2

Reply:
Scott added that, this isn't really part of this PR, she just added a comma. It is just part of the documentation, so I think it's fine.